ruma-events/benches/event_deserialize.rs

```diff
@@ -0,0 +1,130 @@
+// In order to run use either `cargo bench` or to save if you
+// want to save the result to compare branches use
+// `cargo bench --bench event_deserialize -- --save-baseline test-output`
+// Since we are in a workspace the default `cargo bench` still picks up
+// the args passed to it to avoid this use the above command.
```

Comment:
I don't understand this comment. What's "the default `cargo bench`"? Also, it's not clear to me what is avoided.

Reply:
I'm guessing this is about the `-- --save-baseline` syntax, but that's just an established convention for unix-style CLI tools – don't process stuff after a ` -- ` as flags / options. We don't need to document that here.

Reply:
`cargo bench` works but if you use criterions suggestion `cargo bench -- --save-baseline <name>` it fails with the error `cargo bench unknown option --save-baseline` I'm assuming this has something to do with the fact that we are in a workspace? I should change it to say the above and that the workaround is
`cargo bench --bench event_deserialize -- --save-baseline <name>`

Reply:
I don't know what that error is about. Does it make a difference whether you run that command from the repo root vs `ruma-events`?

Reply:
Nope at first, I thought it did but it gave me the same error both times. I've had this happen before turning off the bench harness worked the other time I found [this](https://github.com/bheisler/criterion.rs/issues/193) and used the `[lib] harness = false` so I have no idea exactly why it won't work but I'd figure workspaces still being a bit flaky with some things.

Reply:
This is what I have now
```
// `cargo bench` works but if you use `cargo bench -- --save-baseline <name>`
// or pass any other args to it it fails with the error
// `cargo bench unknown option --save-baseline`.
// To pass args to criterion use this form
// `cargo bench --bench <name of the bench> -- --save-baseline <name>`
```